pkgs/by-name/eu/eudic/package.nix

```diff
@@ -0,0 +1,104 @@
+{ fetchurl
+, stdenv
+, autoPatchelfHook
+, makeWrapper
+, lib
+, makeDesktopItem
+, copyDesktopItems
+, fontconfig
+# Dependences of applications declaim
+, libnotify
+, libX11
+, libXScrnSaver
+, libXext
+, libXtst
+, libuuid
+, libsecret
+, xdg-utils
+, xdg-utils-cxx
+, at-spi2-atk
+# Dependences of autoPatchelfHook
+, gtk3
+, alsa-lib
+, libdrm
+, e2fsprogs
+, nss
+, libgpg-error
+, libjack2
+, mesa
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "eudic";
+  version = "13.5.2";
+
+  src = fetchurl {
+    url = "https://www.eudic.net/download/eudic.deb?v=${finalAttrs.version}";
+    hash = "sha256-UPkDRaqWF/oydH6AMo3t3PUT5VU961EPLcFb5XwOXVs=";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    makeWrapper
+    copyDesktopItems
+  ];
+
+  # Depends: libgtk-3-0, libnotify4, libnss3, libxss1, libxtst6, xdg-utils, libatspi2.0-0, libuuid1, libsecret-1-0
```

Comment:
Instead of hardcoding a list here, it would be a better idea to link what the source of this information is. Did you unpack the deb file and read the files in there?

Reply:
Actually, the list here is base on the unpack of deb files in control.tar.xz, which is basically the extra info i provide in PR.

I not sure if i should contains it in package.

```
Package: eudic
Version: 13.5.2
Installed-Size: 155980
Priority: extra
Section: application
License: unknown
Vendor: EUSOFT<help@eudic.net>
Architecture: amd64
Maintainer: EUSOFT<help@eudic.net>
Depends: libgtk-3-0, libnotify4, libnss3, libxss1, libxtst6, xdg-utils, libatspi2.0-0, libuuid1, libsecret-1-0
Homepage: https://www.eudic.net
Description: 欧路软件
```